tool/icon_generator/lib/main.dart

```diff
@@ -0,0 +1,149 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+import 'dart:typed_data';
+import 'dart:ui' as ui;
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'cupertino.dart' as cupertino;
+import 'material.dart' as material;
+
+const String root =
+    '/Users/devoncarew/projects/intellij/flutter-intellij/resources/flutter';
```

Comment:
make this a relative path if possible.

Reply:
Yeah, this is a bit of a wart. I'm not 100% sure how to get any path info into the Flutter app. I don't think the cwd for the iOS Simulator process is useful info here, but I'll double check.

Reply:
cwd is `/` :(